src/screens/ChooseTechnology.tsx

```diff
@@ -39,7 +39,7 @@ const ChooseTechnology: React.FC<ScreenProps> = ({ navigation }) => {
 
   const handleListPress = async (index: number) => {
     await localStorage.saveTechnology(technologies[index].id, isChecked);
-    navigation.reset({ index: 0, routes: [{ name: "Home" }] });
+    navigation.reset({ index: 0, routes: [{ name: "ChooseRole" }] });
```

Comment:
Isso tem de ser condicional, porque quando o app abre, se não há uma tecnologia selecionada, essa tela deve ser exibida e depois a `Home`, como era.

Como dá pra fazer: quando for pra `ChooseTecnology` a parte da tela de execução, passar um parâmetro (via `navigation`/`params`) que indica que a seleção de tecnologia é pra uma atividade, e verifica esse parâmetro aqui nessa função.

Reply:
Tem mais coisa nisso, agora não sei se não faz parte da integração de tudo; talvez valha a pena só adicionar esse parâmetro nessa task (pra que essa task não fique muito grande, eu acho) e depois, na integração, fazer o resto para começar a executar uma atividade.